tests/platform_tests/test_advanced_reboot.py

```diff
@@ -21,14 +22,15 @@ def test_fast_reboot(request, get_advanced_reboot):
     advancedReboot.runRebootTestcase()
 
 @pytest.mark.usefixtures('get_advanced_reboot')
+@pytest.mark.device_type('vs')
```

Comment:
>device_type [](start = 13, length = 11)

filter by platfrom kvm?

Reply:
The device_type marker is currently used in other tests to mark kvm-compatible tests [ref: #1860]. I wanted to use the same marker to make it consistent with other test cases.